Makefile

```diff
@@ -173,6 +173,26 @@ swagger: config/swagger.yaml $(SWAGGER_CODEGEN_CLI)
 swagger-docs:
 	(cd ./apps/aehttp && $(MAKE) swagger-docs);
 
+SWAGGER_STATUS = $(shell cd /tmp/epoch-api-docs/ && git status --porcelain)
+
+swagger-publish-docs-init:
+	git clone git@github.com:aeternity/epoch-api-docs.git /tmp/epoch-api-docs
+	cp apps/aehttp/priv/swagger-docs/swagger.json /tmp/epoch-api-docs/
```

Comment:
I don't think the swagger.json is enough, about about the generated documentation itself? Am I missing something?

Reply:
based on comment in pivotal task

```
@dimitarivanov
Once the files are installed, we don't have to update them, except for swagger.json. It has all the information required for the docs and we should sync it.```

Reply:
Seems legit, I missed that 👍